dataflow/manual/content.tex

```diff
@@ -1803,9 +1803,14 @@ \subsection{Interaction of the Checker Framework and the Dataflow Analysis}
 
 \end{itemize}
 
+The \code{AnnotatedTypeFactory} and helper classes need to be prepared to work
+even when dataflow analysis is not finished yet. Code should either check
+whether dataflow analysis is still in progress or handle possible null values.
```

Comment:
Can you state how to check this?  That will make the advice actionable.  I searched for "progress" and "finished" and "complete" and "done" and didn't find anything in the manual, so I don't think readers will be able to figure it out.

Reply:
org.checkerframework.dataflow.analysis.Analysis#isRunning